include/highfive/H5Easy.hpp

```diff
@@ -0,0 +1,212 @@
+/*
+ *  Copyright (c), 2017, Adrien Devresse <adrien.devresse@epfl.ch>
+ *
+ *  Distributed under the Boost Software License, Version 1.0.
+ *    (See accompanying file LICENSE_1_0.txt or copy at
+ *          http://www.boost.org/LICENSE_1_0.txt)
+ *
+ */
+#ifndef H5EASY_HPP
+#define H5EASY_HPP
+
+#include <iostream>
+#include <string>
+#include <vector>
+
+#include "H5DataSet.hpp"
+#include "H5DataSpace.hpp"
+#include "H5DataType.hpp"
+#include "H5File.hpp"
+
+// optionally enable plug-in xtensor and load the library
+#ifdef XTENSOR_VERSION_MAJOR
+#define HIGHFIVE_XTENSOR
+#endif
+
+#ifdef HIGHFIVE_XTENSOR
+#include <xtensor/xarray.hpp>
+#include <xtensor/xtensor.hpp>
+#endif
+
+// optionally enable plug-in Eigen and load the library
+#ifdef EIGEN_WORLD_VERSION
+#define HIGHFIVE_EIGEN
+#endif
+
+#ifdef HIGHFIVE_EIGEN
+#include <Eigen/Eigen>
+#endif
+
+namespace HighFive {
+
+///
+/// Write mode for DataSets
+///
+enum class Mode
```

Comment:
DumpMode

Reply:
This I can live with. I will update the PR.